instance/tests/models/test_openedx_theme_mixins.py

```diff
@@ -90,6 +91,8 @@ def test_colors_and_images_applied(self):
             parsed_vars = yaml.load(variables, Loader=yaml.SafeLoader) or {}
             expected_settings = {
                 'SIMPLETHEME_ENABLE_DEPLOY': True,
+                'EDXAPP_COMPREHENSIVE_THEME_SOURCE_REPO': settings.SIMPLE_THEME_SKELETON_THEME_LEGACY_REPO,
```

Comment:
Do we need to add these values to the CircleCi variables or the defaults will do?

Reply:
@giovannicimolin, yes, we will have to add these variables to CircleCI. But the defaults will also work because the integration tests just look for the presence of the color value in the CSS file.